spec.md

```diff
@@ -843,6 +848,10 @@ message CreateVolumeRequest {
   // VOLUME_ACCESSIBILITY_CONSTRAINTS plugin capability, the SP MAY
   // choose where the provisioned volume is accessible from.
   TopologyRequirement accessibility_requirements = 7;
+
+  // This field is OPTIONAL. This allows the CO to specify the
+  // volume performance class parameters to apply. 
```

Comment:
"volume performance class" is a K8s concept.   Change to "mutable volume parameters"?

Reply:
Modified